src/api/app/models/bs_request_action.rb

```diff
@@ -768,7 +765,7 @@ def check_action_permission!(skip_source = nil)
       # validate group object
       Group.find_by_title!(group_name)
     end
-    if self.role
```

Comment:
As I had removed `role` it was not like that any more, but I'll add `role = nil` again and then it is needed again.